conf/defaults.ini

```diff
@@ -298,6 +298,9 @@ default_home_dashboard_path =
 # Upper limit of data sources that Grafana will return. This limit is a temporary configuration and it will be deprecated when pagination will be introduced on the list data sources API.
 datasource_limit = 5000
 
+# Upper limit of bytes in the response body after http calls
+response_limit = 1000000
```

Comment:
Put it there as a dummy length - can revisit to put something which actually has a meaning